cookbooks/fb_init/recipes/site_settings.rb

```
@@ -37,9 +37,7 @@
   node.default['fb_postfix']['aliases'][src] = dst
 end
 
-if File.exist?('/etc/postfix/skip_mailgun')
-  Chef::Log.warn("fb_init: Skipping mailgun postfix setup!")
-else
+if File.exist?('/etc/sasl_passwd')
```

I don't want to silently fail on this.
If /etc/postfix/skip_mailgun exists, then skip mailgun that's fine, someone requested it.

Otherwise, if /etc/sasl_passwd doesn't exist, fail hard.